pkg/cloudprovider/suite_test.go

```diff
@@ -134,6 +136,7 @@ var _ = AfterEach(func() {
 })
 
 var _ = Describe("CloudProvider", func() {
+	It("should delete the nodeclaim if we spot a vm without the extensions in the cache", func() {})
```

Comment:
Missing an actual test body?

Reply:
oh i wrote this test somewhere else let me remove it